src/main/java/com/vesta/service/UserService.java

```diff
@@ -0,0 +1,22 @@
+
+package com.vesta.service;
+
+import com.vesta.repository.UserRepository;
+import com.vesta.service.converter.UserConverter;
+import com.vesta.service.dto.UserDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserConverter userConverter;
+
+    public UserDto getById(Long id) {
+        return userConverter.converter(userRepository.findById(id).orElseThrow(null));
```

Comment:
orElseThrow is used to throw an exception, in your case you need to use simple orElse